--- builddeb.orig	2020-08-29 10:41:02.868288699 -0400
+++ builddeb	2020-08-29 10:41:50.952288699 -0400
@@ -71,7 +71,8 @@
 	installed_image_path="boot/vmlinux-$version"
 	;;
 *)
-	installed_image_path="boot/vmlinuz-$version"
+	installed_image_path="boot/kernel$(echo $version | sed 's/[^-]*-v\?\([^-]*\).*asl+/\1/g').img"
+	echo $installed_image_path
 esac
 
 BUILD_DEBUG=$(if_enabled_echo CONFIG_DEBUG_INFO Yes)
@@ -90,8 +91,10 @@
 	cp $KCONFIG_CONFIG "$tmpdir/usr/share/doc/$packagename/config"
 	gzip "$tmpdir/usr/share/doc/$packagename/config"
 else
-	cp System.map "$tmpdir/boot/System.map-$version"
-	cp $KCONFIG_CONFIG "$tmpdir/boot/config-$version"
+	cp arch/arm/boot/dts/*.dtb "$tmpdir/boot/"
+	mkdir "$tmpdir/boot/overlays"
+	cp arch/arm/boot/dts/overlays/*.dtb* "$tmpdir/boot/overlays/"
+	cp arch/arm/boot/dts/overlays/README "$tmpdir/boot/overlays/"
 fi
 cp "$($MAKE -s -f $srctree/Makefile image_name)" "$tmpdir/$installed_image_path"
 
@@ -164,6 +167,8 @@
 	chmod 755 "$tmpdir/DEBIAN/$script"
 done
 
+../recompile_scripts.sh
+
 # Build kernel header package
 (cd $srctree; find . -name Makefile\* -o -name Kconfig\* -o -name \*.pl) > "$objtree/debian/hdrsrcfiles"
 (cd $srctree; find arch/*/include include scripts -type f -o -type l) >> "$objtree/debian/hdrsrcfiles"
